com.jeffser.Alpaca.Plugins.AMD.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "com.jeffser.Alpaca.Plugins.AMD",
+    "branch": "1.0",
```

Comment:
well everytime Alpaca gets updated with a new version of Ollama I would also need to add a new version of rocm, I think I should leave it versioned

Reply:
that's not what these "version" are about. There are branches so if you really need an atomic update it should be part of the main package. And yes you can build an extension out of an existing package.

Reply:
what do you mean by making in part of the main package? what I want to do is make this thing optional for just the people who need ROCm support, would that work in that case? 

And about the versioning thing, what I mean is that the Ollama instance that comes with the main package should always match this AMD extension, I believe it doesn't work if they aren't a match

Reply:
> what do you mean by making in part of the main package? 

https://docs.flatpak.org/en/latest/extension.html#bundled-extensions

Reply:
>  what I mean is that the Ollama instance that comes with the main package should always match this AMD extension, I believe it doesn't work if they aren't a match

Yes but you aren't providing multiple branches of your app. So there is going to be only one Ollama instance present, ie. whatever the latest update of the app has.

So the extension doesn't need to have separate branches.

If you drop `branch` here and here https://github.com/flathub/com.jeffser.Alpaca/blob/a0db0dfca119403fada0604908807da80f631a54/com.jeffser.Alpaca.json#L29 it will default to `stable`.

Reply:
I removed the branch and version line in the manifests